tensorflow/compiler/tf2tensorrt/convert/convert_nodes.cc

```diff
@@ -2254,10 +2250,11 @@ tensorflow::Status ConvertStridedSliceHelper(OpConverterParams* params,
     }
   }
   if (pad_dims.empty()) {
-    // No dimensions are changed. We could create a padding layer anyway with
-    // values of 0.
+    // No dimensions are changed. Create a no-op layer so tests don't break.
     if (params->validation_only) return Status::OK();
-    params->outputs->push_back(inputs.at(0));
+    nvinfer1::IShuffleLayer* layer = params->converter->network()->addShuffle(
```

Comment:
Just curious: why do we need to add this no-op layer? Does adding the input tensor as output work?

Reply:
The C++ unit tests crash in the case that no layer is created, because an engine cannot be empty. Perhaps it would be better if I improved the capability of the unit tests?

Reply:
Agreed, could you add a comment about this and a TODO?